tf-modules/ingest/message_template.tf

```diff
@@ -31,7 +31,9 @@ data "aws_iam_policy_document" "lambda_processing_role_get_secrets" {
     actions   = ["secretsmanager:GetSecretValue"]
     resources = [
       aws_secretsmanager_secret.message_template_cmr_password.arn,
-      aws_secretsmanager_secret.message_template_launchpad_passphrase.arn
+      aws_secretsmanager_secret.message_template_launchpad_passphrase.arn,
```

Comment:
Nit: this resource being in the file is slightly more confusing if it's no longer specific to just message template secrets

Reply:
Excellent point. 

73bc106b5

Still, not perfect as the launchpad passphrase is used both for the template *and* api auth..